test/test/helpers/use-sinon/index.js

```diff
@@ -43,15 +43,17 @@ export function useFakeTimers( now = 0, clockCallback = noop ) {
  *
  * @param  {Object|Function} config The configuration to use, or a callback that is invoked with the sandbox instance
  * @param  {Function} sandboxCallback A callback function that is invoked with the sandbox instance
+ * @returns { {sandbox: Object} } Object with .sandbox property that points to the current sandbox instance
  */
 export function useSandbox( config, sandboxCallback = noop ) {
+	const wrapper = {};
 	if ( isFunction( config ) && sandboxCallback === noop ) {
 		sandboxCallback = config;
 		config = undefined;
 	}
 
 	before( function() {
-		this.sandbox = sinon.sandbox.create( config );
+		wrapper.sandbox = this.sandbox = sinon.sandbox.create( config );
```

Comment:
It also binds its props to `this`? I think in order to able to use `test`, we need to use one ES5 function to pass to `sinon.test` and the rest of the functions must be arrow functions to retain the value of `this`. Seems more fragile?